src/harness/fourslash.ts

```diff
@@ -2337,6 +2337,38 @@ namespace FourSlash {
             }
         }
 
+        public verifyCodeFix(options: FourSlashInterface.VerifyCodeFixOptions) {

```

Comment:
Consider merging this into `rangeAfterCodeFix` (possibly in a separate PR) to keep the 4/ API simple.

Reply:
I'm planning to remove all `rangeAfterCodeFix` in another PR.